inginious/frontend/user_manager.py

```diff
@@ -351,7 +376,8 @@ def get_users_info(self, usernames) -> Dict[str, Optional[UserInfo]]:
 
         infos = self._database.users.find({"username": {"$in": remaining_users}})
         for info in infos:
-            retval[info["username"]] = UserInfo(info["realname"], info["email"], info["username"], info["bindings"], info["language"])
+            retval[info["username"]] = UserInfo(info["realname"], info["email"], info["username"], info["bindings"],
```

Comment:
BTW, maybe this could also be changed to a dict comprehension. I'm however a bit concerned as this code looks like the new ``get_users`` function you defined. There may be some useless code duplication here.